src/pages/ReimbursementAccount/BusinessInfo/substeps/TypeBusiness.tsx

```diff
@@ -47,20 +46,19 @@ function TypeBusiness({reimbursementAccount, onNext, isEditing}) {
             <Picker
                 inputID={companyIncorporationTypeKey}
                 label={translate('businessInfoStep.companyType')}
-                items={_.map(_.keys(CONST.INCORPORATION_TYPES), (key) => ({value: key, label: translate(`businessInfoStep.incorporationType.${key}`)}))}
+                items={Object.keys(CONST.INCORPORATION_TYPES).map((key) => ({value: key, label: translate(`businessInfoStep.incorporationType.${key}` as TranslationPaths)}))}
```

Comment:
We can narrow it down a little. You can create a type for `keyof typeof CONST.INCORPORATION_TYPES`
for example 
`type IncorporationType = keyof typeof CONST.INCORPORATION_TYPES;`
```suggestion
                items={Object.keys(CONST.INCORPORATION_TYPES).map((key) => ({value: key, label: translate(`businessInfoStep.incorporationType.${key as IncorporationType}`)}))}
```